aws/resource_aws_iam_group_policy_test.go

```diff
@@ -101,6 +110,11 @@ func TestAccAWSIAMGroupPolicy_namePrefix(t *testing.T) {
 					testAccCheckAWSIAMGroupPolicyNameMatches(&groupPolicy1, &groupPolicy2),
 				),
 			},
+			{
+				ResourceName:      "aws_iam_group_policy.test",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
```

Comment:
One minor fix required here:

```
--- FAIL: TestAccAWSIAMGroupPolicy_namePrefix (18.56s)
    testing.go:568: Step 2 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) {
        }
        
        
        (map[string]string) (len=1) {
         (string) (len=11) "name_prefix": (string) (len=24) "test-7166041588452991103"
        }
```

At some point in the future we may make the resources `XXX_prefix` "aware" during import by checking for the format of the name, etc. value for the dash and number of numeric characters that would match our generated values from these prefix fields, but for now these prefix attribute only exist within Terraform (and not the API to refresh during `Read`) so we can ignore the difference on import for now.

```suggestion
				ImportStateVerifyIgnore: []string{"name_prefix"},
			},
```